src/test/java/org/openstreetmap/atlas/checks/validation/tag/ImproperAndUnknownRoadNameCheckTest.java

```diff
@@ -53,6 +53,19 @@ public void testConfigValid()
         Assert.assertEquals(0, flagCount);
     }
 
+    @Test
+    public void testInvalidEdgeIdentifier()
+    {
+        int flagCount = 0;
+        for (final CheckFlag flag : new ImproperAndUnknownRoadNameCheck(
+                ConfigurationResolver.emptyConfiguration())
+                        .flags(this.setup.inValidEdgeIdentifier()))
+        {
+            flagCount++;
+        }
```

Comment:
Nit: this could be condensed by using a ConsumerBasedExpectedCheckVerifier, or Iterables.count().